platform/android/api/api.cpp

```diff
@@ -30,10 +30,11 @@
 
 #include "api.h"
 
-#include "core/config/engine.h"
 #include "java_class_wrapper.h"
 #include "jni_singleton.h"
```

Comment:
> Includes from the platform port or module should be included with relative paths (relative to the root folder of the modular component, e.g. `platform/linuxbsd/`)

Should this then be?..

```suggestion
#include "api/java_class_wrapper.h"
#include "api/jni_singleton.h"
```

Reply:
Indeed!

Reply:
Fixed.